yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```

Comment:
Is this lock file supposed to be tracked/committed in?

Reply:
I think it typically is. https://yarnpkg.com/lang/en/docs/yarn-lock/#toc-check-into-source-control